content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1790,6 +1790,26 @@ uses this annotation.
 See [annotations](https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/guide/service/annotations/)
 in the AWS load balancer controller documentation.
 
+### service.beta.kubernetes.io/aws-load-balancer-security-groups (deprecated) {#service-beta-kubernetes-io-aws-load-balancer-security-groups}
+
+Example: `service.beta.kubernetes.io/aws-load-balancer-security-groups: "sg-53fae93f, sg-8725gr62r"`
+
+Used on: Service
+
+The cloud controller manager integration with AWS elastic load balancing configures
+this annotation to specify a comma seperated list of frontend securityGroups you want
+to attach to an NLB. Both name and ID of securityGroups are supported where name matches
+a `Name` tag, not the `groupName` attribute.
```

Comment:
Please document the actual current use of this annotation, which I believe is by the AWS Load Balancer controller.

However, if you do not know the right answer, you should very clearly mention in the PR description about which details may not be right or are relying on guesses.

Informed guesses are OK, but you should be looking at source code, other documentation etc.

Reply:
Noted. 
Also would like to leave the reference link of official was page for some of the information mentioned here: https://aws.amazon.com/blogs/containers/network-load-balancers-now-support-security-groups/